content/learn/contribute/helping-out/creating-examples.md

```diff
@@ -29,7 +58,13 @@ Please adhere to the following guidelines when creating or updating an example.
 ### Organization
 
 1. Examples should live in an appropriate subfolder of `/examples`.
-2. Examples should be a single file if possible.
+   1. API examples live in `examples/api`, then subdivided by engine category.
+   2. Usage examples live in `examples/usage`, then subdivided by domain.
+   3. Game examples live in `examples/games`, with each game example getting its own dedicated subfolder.
+   4. Stress tests live in the root level `stress-tests` folder.
+   5. Testbeds live in the root level `tests` folder, subdivided by domain (e.g. `3d`).
+2. Each example should consist of exactly one `.rs` file, except where assets or shaders are required.
+   1. Game examples are an exception to this, and should be structured as self-contained small projects with a realistic file and module structure.
```

Comment:
This was a huge reason for why I didn't seriously consider contributing a tiny game before, so this is *very* welcome :)